src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/client/gl/MixinGlDebug.java

```diff
@@ -32,10 +32,15 @@ private void injectAdvancedDebugInfo(Logger logger, String format, Object arg) {
             }
         }
 
-        String locationText = "?";
+        StringBuilder locationText = new StringBuilder();
+        locationText.append('?');
 
         if (finalElement != null) {
-            locationText = finalElement.getClassName() + "." + finalElement.getMethodName() + ":" + finalElement.getLineNumber();
+            locationText.append(finalElement.getClassName())
+                    .append('.')
+                    .append(finalElement.getMethodName())
+                    .append(':')
+                    .append(finalElement.getLineNumber());
         }
```

Comment:
This is not equivalent to the original code.

This now prints `(at ?ClassName.method:137)`.